rock_paper_scissors.py

```diff
@@ -97,11 +97,14 @@ def rounds(self):
 
     @rounds.setter
     def rounds(self, value):
-        if value % 2 != 0:
-            self._rounds = value
-        else:
+        try:
+            value = int(value)
+            if value % 2 == 0 or value < 0:
+                raise ValueError
```

Comment:
I also like the clarity of reading of your suggested approach.
Where would be a good spot to assign the value to the games rounds?
`self._rounds = value` 

After doing some reading, I was under the impression that when catching exceptions, you would handle them in the same try statement, and the else statement executed if no exceptions occurred, hence the route I took.